lua/obsidian/picker_utils.lua

```diff
@@ -16,12 +16,26 @@ M.telescope_mappings = function(map, client, initial_query)
       require("telescope.actions").close(prompt_bufnr)
       client:command("ObsidianNew", { args = query })
     end,
+
+    obsidian_insert_link = function(prompt_bufnr)
+      require("telescope.actions").close(prompt_bufnr)
+      local path = require("telescope.actions.state").get_selected_entry().path
+      local note = require("obsidian").Note.from_file(path, client.dir)
+      local link = client:format_link(note, {})
```

Comment:
Had to pass {} to the format_link function to prevent a nil exception. Maybe this parameter should be made optional in the format_link function.